main.cpp

```diff
@@ -1,3 +1,8 @@
+#include <iostream>
+
+#include <iostream>
+#include <string.h>
```

Comment:
inclusion of deprecated C++ header 'string.h'; consider using 'cstring' instead [modernize-deprecated-headers]